lib/minitest/parallel_fork/ctrl_c.rb

```diff
@@ -5,30 +5,35 @@ module Minitest::ParalleForkCtrlC
   include Minitest::Halt
 
   def run_before_parallel_fork_hook
+    @force_exit = false
     Signal.trap(:INT) do
-      puts "\nInterrupted."
-      puts "Exiting ..."
-      at_exit do
-        exit 130
-      end
-      children_pids.each do |pid|
-        children_pids.each do |pid|
-          begin
-            Process.kill('USR1', pid)
-          rescue Errno::ESRCH
-            # Process already terminated
-          end
+      if @force_exit
+        child_pids.each do |pid|
```

Comment:
We should probably DRY up this and the loop below by adding a method that takes the signal, and sends it to each pid.

There also is a naming difference here `child_pids` vs. `children_pids` below.   However, if you adjust for the recommendations below, you can use `@parallel_fork_child_pids`.